webapp/src/components/card/subscription/index.tsx

```diff
@@ -76,10 +76,30 @@ const SubscriptionCard = ({handleDeleteSubscrption, subscriptionDetails: {channe
                             </div>
                             <div className='card-chip-wrapper'>
                                 {
-                                    repositoryName && <Chip text={`Repository is ${repositoryName}`}/>
+
+                                    // Remove the extra character "/" from start and end of the area path string returned by the API
+                                    areaPath && <Chip text={`Area path - ${areaPath.substring(1, areaPath.length - 1)}`}/>
+                                }
+                                {
+                                    repositoryName && <Chip text={`Repository is: ${repositoryName}`}/>
+                                }
```

Comment:
I think writing these in one line would be better in terms of readability and will also reduce the LOC.